addons/website_crm_score/models/pageview.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+from openerp import fields, models, SUPERUSER_ID, tools
+from psycopg2 import IntegrityError
+
+
+class pageview(models.Model):
+    _name = "website.crm.pageview"
+
+    view_date = fields.Datetime(string='Viewing Date')
+    lead_id = fields.Many2one('crm.lead', string='Lead')
+    partner_id = fields.Many2one('res.partner', string='Partner')
+    url = fields.Char(string='Url')
+
+    def create_pageview(self, cr, uid, vals, context=None, test=False):
+        # returns True if the operation in the db was successful, False otherwise
+        lead_id = vals.get('lead_id', None)
+        partner_id = vals.get('partner_id', None)
+        url = vals.get('url', None)
+        view_date = fields.Datetime.now()
+
+        with self.pool.cursor() as pv_cr:
```

Comment:
because cr.execute can fail and is not critical...
It's a nice to have information that cannot break other things